packages/core/src/codewhisperer/views/activeStateController.ts

```diff
@@ -53,9 +61,9 @@ export class ActiveStateController implements vscode.Disposable {
 
     private _isReady: boolean = false
 
-    private onReady(): void {
+    private async onReady(): Promise<void> {
         this._isReady = true
-        this._refresh(vscode.window.activeTextEditor)
+        await this._refresh(vscode.window.activeTextEditor)
```

Comment:
If you _intentionally_ want this to be async (i.e. "don't care if it fails"), an alternative is to `.catch()` without `await`.
```suggestion
       this._refresh(vscode.window.activeTextEditor).catch(... /* log the error */)
```